donut/modules/editor/templates/editor_page.html

```diff
@@ -1,27 +1,42 @@
 {% extends "layout.html" %}
 {% block page %}
+<head>
 <link rel="stylesheet" href={{ url_for('editor.static', filename='style.css')}}>
-
+  <script src="https://cdn.rawgit.com/showdownjs/showdown/1.0.1/dist/showdown.min.js"></script>
+  <script src={{ url_for('editor.static', filename='script.js', _external = True)}} type="text/javascript"></script>
+  <script type="text/javascript">
+  $SCRIPT_ROOT = {{ request.script_root|tojson|safe }};
+  </script>
+</head>
```

Comment:
As above, `<link rel="stylesheet">` should go in `{% block styles %}` and `<script>` should go in `{% block scripts %}`